src/contentful/index.ts

```diff
@@ -73,16 +73,29 @@ export const normalize = (
       total?: number
       limit?: number
       skip?: number
+      pages: number
+      page: number
+      hasNextPage?: boolean
+      hasPrevPage?: boolean
     }
   }
   data: Array<unknown> | unknown
 } => {
+  const hasNextPage = content.limit + content.skip < content.total
+  const hasPrevPage = content.skip > 0
+  const currentPage =
+    'skip' in content && 'limit' in content ? content.skip / content.limit + 1 : undefined
   return {
     meta: {
       pagination: {
         ...('limit' in content && { limit: content.limit }),
         ...('total' in content && { total: content.total }),
         ...('skip' in content && { skip: content.skip }),
+        ...('limit' in content &&
```

Comment:
Is there a case in which 'limit' and 'total' does not exist in the content, or is this just so that TS does not complain? If there is indeed a case in which these ones are undefined, should we do the same check when we compute the 'hasNextPage'?

Reply:
Mostly TS, I haven't noticed a case where any of the params are missing